modules/xfem/test/tests/bimaterials/glued_bimaterials_2d.i

```diff
@@ -0,0 +1,298 @@
+  [./a_strain_xx]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 0
+    index_j = 0
+    variable = a_strain_xx
+  [../]
+  [./a_strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 1
+    index_j = 1
+    variable = a_strain_yy
+  [../]
+  [./a_strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = A_total_strain
+    index_i = 0
+    index_j = 1
+    variable = a_strain_xy
+  [../]
+  [./b_strain_xx]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 0
+    index_j = 0
+    variable = b_strain_xx
+  [../]
+  [./b_strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 1
+    index_j = 1
+    variable = b_strain_yy
+  [../]
+  [./b_strain_xy]
+    type = RankTwoAux
+    rank_two_tensor = B_total_strain
+    index_i = 0
+    index_j = 1
+    variable = b_strain_xy
+  [../]
+[]
+
+[Constraints]
+  [./dispx_constraint]
+    type = XFEMSingleVariableConstraint
+    use_displaced_mesh = false
+    variable = disp_x
+    alpha = 1e8
+  [../]
+  [./dispy_constraint]
+    type = XFEMSingleVariableConstraint
+    use_displaced_mesh = false
+    variable = disp_y
+    alpha = 1e8
+  [../]
+[]
+
+[BCs]
+  [./bottomx]
+    type = PresetBC
+    boundary = bottom
+    variable = disp_x
+    value = 0.0
+  [../]
+  [./bottomy]
+    type = PresetBC
+    boundary = bottom
+    variable = disp_y
+    value = 0.0
+  [../]
+  [./topx]
+    type = FunctionPresetBC
+    boundary = top
+    variable = disp_x
+    function = 0.03*t
+  [../]
+  [./topy]
+    type = FunctionPresetBC
+    boundary = top
+    variable = disp_y
+    function = '0.03*t'
+  [../]
+[]
+
+[Materials]
+  [./elasticity_tensor_A]
+    type = ComputeIsotropicElasticityTensor
+    base_name = A
+    youngs_modulus = 1e9
+    poissons_ratio = 0.3
+  [../]
+  [./strain_A]
+    type = ComputeSmallStrain
+    base_name = A
+  [../]
+  [./stress_A]
+    type = ComputeLinearElasticStress
+    base_name = A
+  [../]
+  [./elasticity_tensor_B]
+    type = ComputeIsotropicElasticityTensor
+    base_name = B
+    youngs_modulus = 1e5
+    poissons_ratio = 0.3
+  [../]
+  [./strain_B]
+    type = ComputeSmallStrain
+    base_name = B
+  [../]
+  [./stress_B]
+    type = ComputeLinearElasticStress
+    base_name = B
+  [../]
+  [./combined]
+    type = LevelSetBiMaterialStress
+    levelset_plus_base = 'A'
+    levelset_minus_base = 'B'
+    level_set_name = ls
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+
+  solve_type = 'PJFNK'
+  petsc_options_iname = '-ksp_gmres_restart -pc_type -pc_hypre_type -pc_hypre_boomeramg_max_iter'
+  petsc_options_value = '201                hypre    boomeramg      8'
+
+  line_search = 'bt'
+
+# controls for linear iterations
+  l_max_its = 20
+  l_tol = 1e-3
+
+# controls for nonlinear iterations
+  nl_max_its = 15
+  nl_rel_tol = 1e-6
+  nl_abs_tol = 1e-5
+
+# time control
+  start_time = 0.0
+  dt = 0.1
+  num_steps = 2
+
+  max_xfem_update = 1
```

Comment:
What does this parameter do?

Reply:
Good question. We limit the xfem to update only once per time step.